server/src/inmantals/server.py

```diff
@@ -627,7 +626,7 @@ async def textDocument_hover(self, textDocument, position):
 
         data = list(range)[0].data
         docstring = textwrap.dedent(data.docstring.strip("\n")) if data.docstring else ""
-        docstring = docstring.replace(" ", "&nbsp;")
+        docstring = docstring.replace(" ", "&nbsp;").replace("\n", "\n\n")
```

Comment:
I noticed the formatting was not right some time. I took the liberty to add this fix to the PR. 

before:

![Screenshot from 2023-04-28 14-06-51](https://user-images.githubusercontent.com/95339363/235143717-789c4d6e-ccf1-4520-a99e-c9dcd5573519.png)

after:
![Screenshot from 2023-04-28 14-08-49](https://user-images.githubusercontent.com/95339363/235143729-cda46cae-26a8-4455-8364-ade084080813.png)